drivers/income_benefits_report/app/models/income_benefits_report/stayer_households.rb

```diff
@@ -38,7 +38,7 @@ def stayer_households_data
           income_relation: :later_income_record,
         },
         stayers_adults_with_any_income_at_entry: {
-          title: 'Total Adults with any income at Entry',
+          title: 'Total Adults with any Income at Entry',
```

Comment:
@awisema , I'm not sure what's up with GitHub's diff logic, these income_benefits_report changes are all on `pre-release` already.